views/layout.pug

```diff
@@ -23,13 +23,16 @@ html
   - var includefooter = true;
   body(data-developer-key=developerKey,data-thingpedia-url=Config.THINGPEDIA_URL,data-base-url=Config.BASE_URL,data-csrf-token=csrfToken)
     nav.navbar-inverse.navbar.navbar-fixed-top(role="banner")
+      
       div.navbar-header
         button(class="navbar-toggle collapsed", data-toggle="collapse", data-target="#page-nav")
           span.sr-only Toggle navigation
           span.icon-bar
           span.icon-bar
           span.icon-bar
-        span.navbar-brand.scroller.hidden-xs.hidden-sm Almond
+        span.navbar-brand.scroller.hidden-xs.hidden-sm 
+          img.logo_top(src=Config.BASE_URL + '/images/almond.png')
```

Comment:
There is something wrong with the logo when I tried in Firefox, it overflows the navbar.

Reply:
> do you mean usability accessibility?

Yeah I mean accessibility (a11y) - as in, accessible to people with disabilities.

> design choice, it is not the same "content", it lead to a different path than the skills box

It looks off to me. Maybe it's just me.